app_bikes/models.py

```diff
@@ -20,13 +20,20 @@ class AppBikes(App):
     bikes = models.PositiveIntegerField(_L('Nombre de vélos disponibles'), null=True)
     status = models.NullBooleanField(_L('En fonctionnement ?'), null=True)
 
+    def should_update(self):
+        if self.last_activity is None:
+            return True
```

Comment:
I don't think this can happen : https://github.com/bgaultier/laboitepro/blob/5a248d4a46f08d98eab21458c21dc54a2d8a1518/boites/models.py#L23

As soon as the model is saved, `self.last_activity` will be set to the current time.